Lib/CakeMigration.php

```diff
@@ -602,6 +602,9 @@ protected function _clearCache() {
 		DboSource::$methodCache = array();
 		$keys = Cache::configured();
 		foreach ($keys as $key) {
+			if ($key == 'session') {
+				continue;
+			}
```

Comment:
We will likely want to add a `_clearSession()` function that we can call from within a migration itself when we need to force a session clear from a migration.